### PR TITLE
Add view on provider site to right click menu

### LINKED ISSUE #446: 'View on Curse'  in right click menu.
**Is your feature request related to a problem? Please describe.**
If i want to view the changelog of an addon I first click the addon to get the dialog showing the info, then click the view on curse, and finally close the intermediate dialog window

**Describe the solution you'd like**
add "view on curse" to the right click menu.
This would streamline the method to access the website

**Describe alternatives you've considered**
alternatively, the external link icon could be added tot he provider column or another column on the addon list

**Additional context**
none

### PR DESCRIPTION
**Pull request by stevietv**
fixes #446 by adding a link in the right click menu

### CODE REVIEW

**Comment by stevietv** [comment]
fixes #446 by adding a link in the right click menu

File: wowup-electron/src/app/pages/my-addons/my-addons.component.html [hunk 1]
```diff
@@ -227,6 +227,9 @@ <h1>{{ "COMMON.SEARCH.NO_ADDONS" | translate }}</h1>
     <button mat-menu-item (click)="onShowFolder(listItem.addon)">
       {{ "PAGES.MY_ADDONS.ADDON_CONTEXT_MENU.SHOW_FOLDER" | translate }}
     </button>
+    <button mat-menu-item (click)="onVisitProvider(listItem.addon)">
+      {{ "DIALOGS.ADDON_DETAILS.VIEW_ON_PROVIDER_PREFIX" | translate }} {{listItem.addon.providerName}}
```
Comment (linaori) [inline wowup-electron/src/app/pages/my-addons/my-addons.component.html:231]: Can you add this translation to `en.json` and then run `npm run i18n` to sync the translations?

**Review by linaori — CHANGES_REQUESTED** [review]

File: wowup-electron/src/app/pages/my-addons/my-addons.component.html (hunk 1 above)
Comment (stevietv) [reply wowup-electron/src/app/pages/my-addons/my-addons.component.html:231]: Its already in the translations as its used on the addon detail popup also.

**Review by linaori — APPROVED** [review]

File: wowup-electron/src/app/pages/my-addons/my-addons.component.html [hunk 2]
```diff
@@ -227,6 +227,9 @@ <h1>{{ "COMMON.SEARCH.NO_ADDONS" | translate }}</h1>
     <button mat-menu-item (click)="onShowFolder(listItem.addon)">
       {{ "PAGES.MY_ADDONS.ADDON_CONTEXT_MENU.SHOW_FOLDER" | translate }}
     </button>
+    <button mat-menu-item (click)="onVisitProvider(listItem.addon)">
```
Comment (jliddev) [inline wowup-electron/src/app/pages/my-addons/my-addons.component.html:230]: Can we drop this click handler and make it the same as the details link? If we make it an a and not a button it should look the same. Note the appExternalLink as well.

https://github.com/WowUp/WowUp/blob/develop/wowup-electron/src/app/components/addon-detail/addon-detail.component.html#L33

File: wowup-electron/src/app/pages/my-addons/my-addons.component.html (hunk 1 above)
Comment (jliddev) [inline wowup-electron/src/app/pages/my-addons/my-addons.component.html:231]: Can we add the same little icon from details for consistency? 

https://github.com/WowUp/WowUp/blob/develop/wowup-electron/src/app/components/addon-detail/addon-detail.component.html#L36

File: wowup-electron/src/app/pages/my-addons/my-addons.component.ts [hunk 3]
```diff
@@ -395,6 +395,15 @@ export class MyAddonsComponent implements OnInit, OnDestroy, AfterViewInit {
     }
   }
 
+    public onVisitProvider(addon: Addon) {
```
Comment (jliddev) [inline wowup-electron/src/app/pages/my-addons/my-addons.component.ts:398]: This can likely be replaced by the appExternalLink directive

**Review by jliddev — CHANGES_REQUESTED** [review]

File: wowup-electron/src/app/pages/my-addons/my-addons.component.html (hunk 2 above)
Comment (stevietv) [reply wowup-electron/src/app/pages/my-addons/my-addons.component.html]: I've converted this button to an a

File: wowup-electron/src/app/pages/my-addons/my-addons.component.html (hunk 1 above)
Comment (stevietv) [reply wowup-electron/src/app/pages/my-addons/my-addons.component.html:231]: icon is added, hope that I resized it appropriately in the css

File: wowup-electron/src/app/pages/my-addons/my-addons.component.ts (hunk 3 above)
Comment (stevietv) [reply wowup-electron/src/app/pages/my-addons/my-addons.component.ts]: method is removed from ts - no longer needed

**Review by jliddev — APPROVED** [review]
Looks good

**Pull request merged**